### DISCUSSION TITLE
I would like to make a translation

### DISCUSSION THREAD

**Discussion by NeonGooRoo** [arianneorpilla/jidoujisho#371] [General]
Hi, I would love to contribute to the app and make a Russian translation.

I have basically 0 real programming experience, I only worked with HTML\CSS, but I think I can figure out how to do just translate all of the UI. 

I see that in the app there is already a way to change the App Locale. Is the feature in the progress? I wanted to just fork the app and change all the text to Russian, but probably adding just the localization into the main app would be a much better option.

**Comment by arianneorpilla** [comment]
Hi @NeonGooRoo,

I really appreciate that you've reached out here. It means a lot to me. You can work on translating the existing app in [**Crowdin**](https://crowdin.com/project/jidoujisho). There is a partial effort to translate the app in Russian there,

Though I have to mention, I am working on something. I will still definitely be able to use keys from Crowdin when the time comes that I can share that work with everyone.

**Reply by NeonGooRoo** [reply]
Amazing! Thank you! 

Your app is insanely good.

**Reply by NeonGooRoo** [reply]
Hi. Me and some other people finished the translation, and I see some other languages are already finished too. When do you plan to make an update with different language support?

**Comment by comradekingu** [comment]
@lrorpilla Any chance you can use Weblate instead of Crowdin?
It is a lot better for privacy, and it is a better tool. :)
